internal/providers/terraform/azure/registry.go

```diff
@@ -236,11 +281,43 @@ var FreeResources []string = []string{
 	"azurerm_container_registry_token",
 	"azurerm_container_registry_webhook",
 
+	// Azure Sentinel
+	"azurerm_sentinel_alert_rule_machine_learning_behavior_analytics",
+	"azurerm_sentinel_alert_rule_fusion",
+	"azurerm_sentinel_alert_rule_ms_security_incident",
+	"azurerm_sentinel_alert_rule_scheduled",
+	"azurerm_sentinel_data_connector_aws_cloud_trail",
+	"azurerm_sentinel_data_connector_azure_active_directory",
+	"azurerm_sentinel_data_connector_azure_advanced_threat_protection",
+	"azurerm_sentinel_data_connector_azure_security_center",
+	"azurerm_sentinel_data_connector_microsoft_cloud_app_security",
+	"azurerm_sentinel_data_connector_microsoft_defender_advanced_threat_protection",
+	"azurerm_sentinel_data_connector_office_365",
+	"azurerm_sentinel_data_connector_threat_intelligence",
+
 	// Azure SQL
 	"azurerm_sql_server",
+	"azurerm_sql_firewall_rule",
+	"azurerm_sql_virtual_network_rule",
+	"azurerm_mssql_firewall_rule",
+
+	// Azure Storage
+	"azurerm_storage_table_entity",
+	"azurerm_storage_management_policy",
+	"azurerm_storage_blob_inventory_policy",
+
+	// Azure Virtual Desktop
```

Comment:
Looking at https://azure.microsoft.com/en-us/pricing/details/virtual-desktop/, I assume the user has to provision the VMs so we'll show costs for those, and there's no additional fee for "configuring" the VMs with desktop apps? Sorry if that's a dump question, I haven't used these resources before...

Reply:
Usually, the Host Pool is where the charges will apply because that's where the "work" is being done. 
These are the infrastructure costs that are not influenced by the amount of Desktop Apps or Remote Apps that are configured.
The other cost part is related to M365/O365 Licensing, which is not showing up in any Azure Bill.

Reply:
I added https://github.com/infracost/infracost/issues/888 for Azure Virtual Desktop